sparrow_tracky/deepsort/tracker.py

```
@@ -61,81 +75,105 @@ def track(self, boxes: FrameBoxes) -> None:
-                    # active -> finished
-                    self.active_tracklets[missing_idx].finalize_missing_boxes()
-                    tracklet_switches[missing_idx] = "finished"
+        
+        # ByteTrack: Separate detections by confidence
+        # Try to get confidences from boxes metadata, default to ones if not available
+        try:
+            confidences = getattr(boxes, 'confidences', None)
+            if confidences is None:
+                # Try to get from metadata
+                confidences = boxes.metadata_kwargs.get('confidences', np.ones(len(boxes)))
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)